<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>元王IDS集成设计仿真平台</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/Top.qrc">
    <normaloff>:/new/wl/Logo_ico.png</normaloff>:/new/wl/Logo_ico.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>891</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>8</horstretch>
      <verstretch>9</verstretch>
     </sizepolicy>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>4</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>主页</string>
      </attribute>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>0</y>
         <width>141</width>
         <height>621</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>111</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pBtnNewTask">
           <property name="text">
            <string>新建任务</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/Now.png</normaloff>:/new/wl/Now.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnInfoInput">
           <property name="text">
            <string>信息输入</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
           <property name="autoRepeatInterval">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnDFRAnalyse">
           <property name="text">
            <string>DFR分析</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnRiskGrade">
           <property name="text">
            <string>风险评分</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnStepJudge">
           <property name="text">
            <string>措施评审</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnStepEdit">
           <property name="text">
            <string>措施修订</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnRiskTable">
           <property name="text">
            <string>风险评估</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnReportOutput">
           <property name="text">
            <string>报告输出</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBtnReportModify">
           <property name="text">
            <string>报告修改</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc/Top.qrc">
             <normaloff>:/new/wl/sure.png</normaloff>:/new/wl/sure.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>5</width>
         <height>621</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtnNext">
       <property name="geometry">
        <rect>
         <x>668</x>
         <y>590</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtnQuit">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>590</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>741</width>
         <height>581</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="FormLayout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Tesk">
    <property name="title">
     <string>任务(&amp;W)</string>
    </property>
    <addaction name="A_SaveNewTask"/>
    <addaction name="A_ViewTask"/>
    <addaction name="A_ViewAllTask"/>
    <addaction name="A_EditTask"/>
    <addaction name="A_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Switch">
    <property name="title">
     <string>切换用户(&amp;X)</string>
    </property>
    <addaction name="A_ChangeUser"/>
   </widget>
   <widget class="QMenu" name="menu_Look">
    <property name="title">
     <string>查看(&amp;Y)</string>
    </property>
    <addaction name="A_RuleQuery"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>外观(&amp;V)</string>
    </property>
    <addaction name="A_TB"/>
    <addaction name="A_SB"/>
   </widget>
   <widget class="QMenu" name="menu_Set">
    <property name="title">
     <string>设置(&amp;Z)</string>
    </property>
    <addaction name="A_EditUser"/>
    <addaction name="A_RightControl"/>
    <addaction name="A_UserManager"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="A_About"/>
   </widget>
   <addaction name="menu_Tesk"/>
   <addaction name="menu_Switch"/>
   <addaction name="menu_Look"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Set"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
  </widget>
  <action name="A_SaveNewTask">
   <property name="text">
    <string>新建任务</string>
   </property>
  </action>
  <action name="A_ViewTask">
   <property name="text">
    <string>查看当前任务</string>
   </property>
  </action>
  <action name="A_ViewAllTask">
   <property name="text">
    <string>查看所有任务</string>
   </property>
  </action>
  <action name="A_EditTask">
   <property name="text">
    <string>编辑任务</string>
   </property>
  </action>
  <action name="A_Quit">
   <property name="text">
    <string>退    出</string>
   </property>
  </action>
  <action name="A_ChangeUser">
   <property name="text">
    <string>切换用户</string>
   </property>
  </action>
  <action name="A_RuleQuery">
   <property name="text">
    <string>规则查询</string>
   </property>
  </action>
  <action name="A_TB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="A_SB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="A_EditUser">
   <property name="text">
    <string>用户信息编辑</string>
   </property>
  </action>
  <action name="A_RightControl">
   <property name="text">
    <string>用户权限控制</string>
   </property>
  </action>
  <action name="A_UserManager">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="A_About">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/Top.qrc"/>
 </resources>
 <connections/>
</ui>
